sieve/sieve.y

```diff
@@ -274,18 +273,13 @@ extern void sieverestart(FILE *f);
 %type <nval> reject
 
 /* enotify - RFC 5435 */
-%token METHOD OPTIONS MESSAGE IMPORTANCE VALIDNOTIFYMETHOD NOTIFYMETHODCAPABILITY
-%token <nval> NOTIFY ENOTIFY ENCODEURL
+%token OPTIONS MESSAGE IMPORTANCE VALIDNOTIFYMETHOD NOTIFYMETHODCAPABILITY
+%token <nval> NOTIFY ENCODEURL
```

Comment:
The ENOTIFY token was never used anywhere and should've been removed from master.  I only noticed it while working on this.

The NOTIFY token remains because the name of the action hasn't changed, just the syntax and capability.